src/main/resources/liquibase/harness/change/expectedSnapshot/edb-e/9.5/alterSequence.json

```diff
@@ -0,0 +1,13 @@
+{
+  "snapshot": {
+    "objects": {
+      "liquibase.structure.core.Sequence": [
+        {
+          "sequence": {
+            "name": "test_sequence"
```

Comment:
add "body" attribute please

Reply:
Liqubase snapshot doesn't capture more information for EDB versions 9.5 and 9.6. Same behavior is true for EDB with Postgres driver